src/components/QuizGeneratorModal.tsx

```diff
@@ -122,196 +122,157 @@ export default function QuizGeneratorModal(): JSX.Element {
     return code;
   };
   return (
```

Comment:
not related to your PR, but for performance reasons I think it's better of `getCode()` was rewritten as `const code = useMemo(..., [quiz])` or something. probably not super important though